### DISCUSSION TITLE
Problem to know the name of the [guilds].tsx

### DISCUSSION THREAD

**Discussion by DoctorPok42** [vercel/next.js#33806] [Help] [answered]
Hello !

I am creating a dashboard for my discord bot only when I create the link to modify a particular server
```tsx
<Link
  href={`../dashboard/${guilds.id}`}>
  <h2>Configure</h2>
</Link>
```

but when i try to get server id in [guilds].tsx page it returns me undefined
```tsx
export const getServerSideProps = async function (
  ctx, params
) {
...
console.log(params);
...
}
```

**Comment by jamesmosier** [comment, marked as answer]
You can get it from `context.params`.

```
export const getServerSideProps = async function (
  ctx
) {
...
console.log(ctx.params);
...
}`
```

[docs for `context` param](https://nextjs.org/docs/api-reference/data-fetching/get-server-side-props#context-parameter)
